actions/commands.go

```diff
@@ -35,33 +35,20 @@ func Commands() {
 	app.Commands = []cli.Command{
 
 		{
-			Name:    "clone",
-			Aliases: []string{"c"},
-			Usage:   "Clone a template from github",
+			Name:  "project",
+			Usage: "Project management",
 			Flags: []cli.Flag{
 				cli.StringFlag{
-					Name:  "branch, b",
-					Value: "master",
-					Usage: "repository branch",
-				},
-				cli.StringFlag{
-					Name:     "destination, d",
-					Required: true,
-					Usage:    "absolute destination file path",
-				},
-				cli.StringFlag{
-					Name:     "owner",
-					Required: true,
-					Usage:    "repository owner",
-				},
-				cli.StringFlag{
-					Name:     "repo",
-					Required: true,
-					Usage:    "repository to download",
+					Name:  "r",
+					Usage: "repository url",
 				},
 			},
 			Action: func(c *cli.Context) error {
-				CloneTemplate(c)
+				if c.NumFlags() == 0 {
+					// ValidateProject()
```

Comment:
Could make this `TODO: Add ValidateProject() function`  to be super clear what is happening (if this is being merged as-is)?